packages/core/strapi/lib/services/entity-service/index.d.ts

```diff
@@ -54,14 +55,19 @@ export interface EntityService {
     pagination: PaginationInfo;
   }>;
 
-  findWithRelationCounts<K extends keyof AllTypes, T extends AllTypes[K]>(
+  findWithRelationCountsPage<K extends keyof AllTypes, T extends AllTypes[K]>(
     uid: K,
     params: Params<T>
   ): Promise<{
     results: T[];
     pagination: PaginationInfo;
```

Comment:
Should this be removed?

Reply:
It was removed in the other function (see comment above)